kanidmd/src/lib/be/idl_sqlite.rs

```diff
@@ -624,22 +575,19 @@ impl IdlSqliteWriteTransaction {
     }
 
     // ! TRACING INTEGRATED
-    pub fn commit(mut self, audit: &mut AuditScope) -> Result<(), OperationError> {
+    pub fn commit(mut self) -> Result<(), OperationError> {
         spanned!("be::idl_sqlite::commit", {
-            lperf_trace_segment!(audit, "be::idl_sqlite::commit", || {
-                // ltrace!(audit, "Commiting BE WR txn");
-                assert!(!self.committed);
-                self.committed = true;
+            // ltrace!(audit, "Commiting BE WR txn");
```

Comment:
Should we add a tracing log for this one?
```suggestion
            trace!("Commiting BE WR txn");
```

Reply:
Added manually.